lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -12,6 +12,15 @@ defmodule ScreenplayWeb.AuthManager do
     "pa-message-admin" => :pa_message_admin
   }
 
+  def max_session_time do
+    Application.get_env(:screenplay, __MODULE__)[:max_session_time]
+  end
+
+  def idle_time do
+    Application.get_env(:screenplay, __MODULE__)[:idle_time]
```

Comment:
Since these are configured at compile time there is not much point in reading them at runtime. We could instead define e.g.

```elixir
@idle_time Application.fetch_env!(:screenplay, [__MODULE__, :idle_time])
```

And this would ensure at compile time that we are, in fact, configuring them as expected.

Reply:
You mean `compile_env!`? Either way, will make that change.